src/models/ItemTableData.ts

```diff
@@ -56,18 +54,18 @@ export default class ItemTableData {
   }
 
   showVolumeColumn(): boolean {
-    return this.items.some((item) => item.volume) && this.isBibPage
+    return this.items.some((item) => item.volume) && !this.inSearchResult
   }
 
   showStatusColumn(): boolean {
-    return this.isBibPage
+    return !this.inSearchResult
   }
 
   showAccessColumn(): boolean {
-    return this.isBibPage && this.isDesktop
+    return !this.inSearchResult
   }
 
   volumeColumnHeading(): string {
-    return this.isArchiveCollection ? "Vol/Date" : "Container"
+    return this.isArchiveCollection ? "Container" : "Vol/Date"
```

Comment:
Why this update? Or where is it documented?

Reply:
I change this because the values were the reverse in DFE:

`const volColumnHeading = isArchiveCollection ? archiveColumnText : volText;`